### DISCUSSION TITLE
Webhook scenario [Oct 07]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4098] [Discord Threads]
> 🙋‍♂️ **prasanth**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### You can conditionally trigger webhook (when a column value is updated to a certain value). custom operations need to be handled by using APIs after receiving webhook  
#  
> 🙋‍♂️ **prasanth**  
#### so what you are proposing is i should setup a webhook api which adds a row in another table and then deletes the row in this table using nocodb apis right?  
#  
> 👨‍🦲 **dstala**  
#### yes, right  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1025364796838776853)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
